src/frontend/AuthProvider.jsx

```diff
@@ -107,7 +107,7 @@ export const KindeProvider = ({children}) => {
         const flags = getClaimValue('feature_flags');
         const flag = flags && flags[code] ? flags[code] : {};
 
-        if (!flag.v && !defaultValue) {
+        if (!flag.v && defaultValue == undefined) {
```

Comment:
Do we need to also change the check on `flag.v` too?  Thinking if the value is `false` or `0`.

Reply:
@coel @peterphanouvong yeah we should. Also realising the JS SDK likely needs the same update